feature-account-api/src/main/java/io/novafoundation/nova/feature_account_api/domain/model/MetaAccount.kt

```diff
@@ -154,6 +154,10 @@ fun MetaAccount.publicKeyIn(chain: Chain): ByteArray? {
     }
 }
 
+fun MetaAccount.multiChainEncryption(): MultiChainEncryption? {
```

Comment:
The name sounds confusing - maybe better to name it something like `substrateMultiChainEncryption`?